src/cmd/group.go

```diff
@@ -291,9 +271,10 @@ var deleteGroupCmd = &cobra.Command{
 }
 
 var importGroupsCmd = &cobra.Command{
-	Use:     "group",
-	Aliases: []string{"groups"},
-	Short:   "Imports groups from a CSV",
+	Use:        "group",
+	Aliases:    []string{"groups"},
+	Short:      "Imports groups from a CSV",
+	Deprecated: `Please convert all Groups into Teams. See https://docs.opslevel.com/docs/groups`,
```

Comment:
Since you removed the create and update functionality - should we also remove the "import" functionality?

Reply:
I'm thinking that customers might use this to get a snapshot of their current Groups so that they can know what's left to convert to Teams

Reply:
Removing per offline discussion